tests/func/test_cache.py

```diff
@@ -164,6 +165,10 @@ def test_abs_path(self):
         self.assertEqual(config["cache"]["dir"], dname.replace("\\", "/"))
 
     def test_relative_path(self):
+        # Fixme
```

Comment:
Let's use consistent comments that are saying that this is temporarily disabled and have a link to the github issue.